source/unified-test-format/unified-test-format.md

```diff
@@ -2134,6 +2134,8 @@ An example of this operation follows:
 Use a `listIndexes` command to check whether the index exists. Note that it is currently not possible to run
 `listIndexes` from within a transaction.
 
+<div id="operation_createEntities">
```

Comment:
Necessary because we previously define a manual anchor for `createEntities` (for the top-level field), and that takes precedence over the natural anchor created from the heading below. And yes, that does break the GitHub-rendered navigation but I'm not trying to fix that here.